src/chrome/content/rules/unicode.org.xml

```diff
@@ -0,0 +1,24 @@
+<ruleset name="unicode.org">
+
+	<target host="unicode.org" />
+	<target host="www.unicode.org" />
+	<target host="cldr-smoke.unicode.org" />
+	<target host="st.unicode.org" />
+	
+<!-- Non-working hosts
+        Redirect to HTTP:
+        	cldr-build.unicode.org
+        	
+        "Secure Connection Failed"
+       		cldr.unicode.org
```

Comment:
Please use a consistent code style (only tabs to indent lines, colon after error type, no quotes)